sp1.ahk

```diff
@@ -156,6 +177,28 @@ AuditionNotesHotkey:
     SendInput {1}
 return
 
+; GabPou's code for quick erase
```

Comment:
I think comment isn't needed here. We have git to track contributers :)

Reply:
Totally forgot to remove that lol